k2/torch/bin/decode.cc

```diff
@@ -0,0 +1,89 @@
+#include <cassert>
+#include <cstdio>
+
+#include "torch/script.h"
+#include "torch/utils.h"
+
+C10_DEFINE_string(jit_pt, "", "Path to exported jit filename.");
+C10_DEFINE_string(feature_pt, "", "Path to pre-computed feature filename.");
+// TODO: add more options
+
+// Load a file saved by torch.save(x, filename), where
+// x can be
+//   - a tensor
+//   - a tuple of tensors
+//   - a list of ints
+//   - a tuple containing: a tensor, a list of ints, etc.
+static torch::IValue Load(const std::string &filename) {
+  FILE *fp = fopen(filename.c_str(), "rb");
+  assert(fp != nullptr);
+
+  fseek(fp, 0, SEEK_END);
+  size_t size = ftell(fp);
+  fseek(fp, 0, SEEK_SET);
+
+  std::vector<char> f(size);
+  size_t len = fread(f.data(), 1, size, fp);
+  assert(len == size);
+
+  fclose(fp);
+
+  return torch::jit::pickle_load(f);
+}
+
+int main(int argc, char *argv[]) {
+  at::set_num_threads(1);
+  at::set_num_interop_threads(1);
+
+  std::string usage = R"(
+    ./bin/decode.py \
```

Comment:
Oh, a typo here. It should be `./bin/decode`

If you clone `k2` and do
```
mkdir build
cd build
cmake ..
make decode
```
you will find a binary in `./bin/decode`